tools/logdump_decompress/logdump_decompress.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+############################################################################
+#
+# Copyright 2025 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+############################################################################
+
+import sys
+import zlib
+from pathlib import Path
+
+
+def read_nbytes(fp, n):
+    data = fp.read(n)
+    if len(data) != n:
+        return None
+    return data
+
+
+def decompress_file(src, dst):
+    with src.open("rb") as fp:
+        while True:
+            len_field = read_nbytes(fp, 4)
+            if len_field is None:      # Normal Exit
+                break
+
+            block_len = int(len_field)
+            comp_data = read_nbytes(fp, block_len)
+            decomp_data = zlib.decompress(comp_data)
+            filtered_data = bytes(b for b in decomp_data if b >= 0x20 or b in (0x0A, 0x0D))
```

Comment:
What do these values mean? 0x0A, 0x0D and 0x20. Could you leave a comment about filtered data?